ios/FluentUI/Controls/IndeterminateProgressBarView.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Indeterminate progress bar. Since iOS's UIprogressView is always determinate, this is created for handling indeterminate scenarios.
+@objc(MSFIndeterminateProgressBarView)
+open class IndeterminateProgressBarView: UIView, ActivityViewAnimating {
+
+    /// The progress bar view should be hidden when animation stops if set to true. The default value is true.
+    @objc open var hidesWhenStopped: Bool = true
+
+    open override var intrinsicContentSize: CGSize {
+        return CGSize(width: UIView.noIntrinsicMetric, height: frame.height)
```

Comment:
We can't be defining the intrinsicContentSize in terms of the current height (frame). One of the fundamental tenets of autolayout is that views/constraints shouldn't depend on their current size and should expect that to be unpredictable

Reply:
Should just be Constants.progressBarHeight?